tests/api-compatibility/reference/.gitignore

```diff
@@ -0,0 +1,2 @@
+Mono.Android.dll
```

Comment:
we should not need this .gitignore, the one in the root is already ignoring *.dll on this folder.

Reply:
Removed, and contents merged with root `.gitignore`.